tests/integration/utils.ts

```diff
@@ -0,0 +1,169 @@
+import { AOS_MODULE_ID, AoClient } from '@ar.io/sdk';
+import AoLoader from '@permaweb/ao-loader';
+import fs from 'node:fs';
+import path from 'node:path';
+
+const __dirname = path.dirname(new URL(import.meta.url).pathname);
+export const TEST_AOS_ANT_WASM = fs.readFileSync(
+  path.join(__dirname, 'fixtures', `aos-ant-${AOS_MODULE_ID}.wasm`),
+);
+
+export const STUB_PROCESS_ID = 'process-id-'.padEnd(43, '1');
+export const STUB_ADDRESS = 'arweave-address-'.padEnd(43, '1');
+export const STUB_ETH_ADDRESS = '0xFCAd0B19bB29D4674531d6f115237E16AfCE377c';
+export const STUB_ANT_REGISTRY_ID = 'ant-registry-'.padEnd(43, '1');
+
+export const AO_LOADER_HANDLER_ENV = {
+  Process: {
+    Id: STUB_ADDRESS,
+    Owner: STUB_ADDRESS,
+    Tags: [
+      { name: 'Authority', value: 'XXXXXX' },
+      { name: 'ANT-Registry-Id', value: STUB_ANT_REGISTRY_ID },
+    ],
+  },
+  Module: {
+    Id: ''.padEnd(43, '1'),
+    Tags: [{ name: 'Authority', value: 'YYYYYY' }],
+  },
+};
+
+export const AO_LOADER_OPTIONS = {
+  format: 'wasm32-unknown-emscripten-metering',
+  inputEncoding: 'JSON-1',
+  outputEncoding: 'JSON-1',
+  memoryLimit: '1073741824', // 1 GiB in bytes
+  computeLimit: (9e12).toString(),
+  extensions: [],
+};
+
+export const DEFAULT_HANDLE_OPTIONS = {
+  Id: ''.padEnd(43, '1'),
+  ['Block-Height']: '1',
+  // important to set the address so that that `Authority` check passes. Else the `isTrusted` with throw an error.
+  Owner: STUB_ADDRESS,
+  Tags: [],
+  Module: 'ANT',
+  Target: STUB_ADDRESS,
+  From: STUB_ADDRESS,
+  Timestamp: Date.now(),
+  // for msg.reply
+  Reference: '1',
+  Data: '1234',
+};
+
+export type HandleFunction = Awaited<ReturnType<typeof AoLoader>>;
+
+export class LocalAO implements Partial<AoClient> {
```

Comment:
nit: add some documentation for this class and how it works/what it does

Reply:
https://github.com/ar-io/ar-io-sdk/pull/387/commits/3a583d7de2f539c0e93d8a13a1112b1d28d8086e